open-craft/frontend-lib-content-components
# feat: video gallery sorting option names

## Description

Change video gallery sorting option names to simpler ones.

Before:
![before](https://github.com/openedx/frontend-lib-content-components/assets/1616648/e420b2cc-49df-43b4-bb17-939921bbe0aa)

After:
![after](https://github.com/openedx/frontend-lib-content-components/assets/1616648/50cd7c0e-8086-45c4-b3eb-054ba6c54e17)

## Testing instructions

1. Open video gallery
1.1. Enable the new video editor
1.2. Create a course unit with a video
1.3. Edit the video (should open in new editor)
1.4. Click "replace video" to go to the gallery
2. Use sorting menu and see the names of sorting options
